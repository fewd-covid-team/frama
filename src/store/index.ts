import { configureStore } from '@reduxjs/toolkit';
import countryReducer, {
  pickRU, pickUA, pickUSA,
} from './slice';

export default configureStore({
  reducer: {
    country: countryReducer,
  },
});

export { pickUA, pickUSA, pickRU };
