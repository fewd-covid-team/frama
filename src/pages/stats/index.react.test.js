import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import StatsScreen from './index';
import store from '../../store';

test('StatsScreen snapshot', () => {
  const component = renderer.create(
    <Provider store={store}>
      <StatsScreen />
    </Provider>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
