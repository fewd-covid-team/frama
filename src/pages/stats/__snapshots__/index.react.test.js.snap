// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatsScreen snapshot 1`] = `
<div>
  <div
    className="sc-bdfBQB fuBMcz"
  >
    <div
      className="sc-gsTEea gEoiSC"
    >
      <button
        aria-label="Pick UA"
        className="sc-dlfnuX dQSMFK"
        onClick={[Function]}
        type="button"
      >
        Ukraine ðŸ‡ºðŸ‡¦
      </button>
       
      <button
        aria-label="Pick USA"
        className="sc-dlfnuX dQSMFK"
        onClick={[Function]}
        type="button"
      >
        USA ðŸ‡ºðŸ‡¸
      </button>
       
      <button
        aria-label="Pick RU"
        className="sc-hKgJUU jUVlOu"
        onClick={[Function]}
        type="button"
      >
        Russia ðŸ‡·ðŸ‡º
      </button>
    </div>
  </div>
  <div>
    <div
      className="sc-eCstlR fFuuXC"
    >
      <p
        className="sc-gKseQn byKqSI"
      >
        Period data
      </p>
      <div
        className="recharts-responsive-container"
        style={
          Object {
            "height": "100%",
            "maxHeight": undefined,
            "minHeight": undefined,
            "minWidth": undefined,
            "width": "100%",
          }
        }
      />
    </div>
  </div>
  <div
    className="sc-jSgvzq khJvwa"
  >
    <p
      className="sc-gKseQn byKqSI"
    >
      Today
    </p>
    <div
      className="recharts-responsive-container"
      style={
        Object {
          "height": "100%",
          "maxHeight": undefined,
          "minHeight": undefined,
          "minWidth": undefined,
          "width": "100%",
        }
      }
    />
  </div>
</div>
`;
